tests/e2e/ctl_v3_completion_test.go

```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build !cov
+// +build !cov
```

Comment:
No need to keep ` +build !cov`, it's just for backward compatibility.
```suggestion
```

Reply:
It isn't a big deal. Please feel free to resolve it in a  separate PR if you want.